src/main/java/mods/eln/sixnode/electricalfiredetector/ElectricalFireDetectorSlowProcess.java

```diff
@@ -69,44 +70,43 @@ public void process(double time) {
             detectionBBCenter.copyFrom(element.getCoordonate());
             switch (element.side) {
                 case XP:
-                    detectionBBCenter.x -= maxRangeHalf;
+                    detectionBBCenter.pos.add(-maxRangeHalf, 0, 0);
```

Comment:
Does this modify the position, or return a new one? Please check.

Reply:
Confirmed that it modifies it. To return a new one you would use pos.setPos(x, y, z).